less/navigation.less

```diff
@@ -148,6 +148,7 @@
           position: absolute;
           left: 101%;
           top: 14px;
+          line-height: 10px;
```

Comment:
Is this any different from changing `top`

Reply:
Not really any different, but it's consistent with the fix that was made to the chevron between site and subsite